### PR TITLE
Added keynote speakers

### PR DESCRIPTION
**Pull request by Davidovich4**

### CODE REVIEW

**Comment by daroczig** [comment]
Thanks a lot @Davidovich4 -- I've added a couple extra icons here and asking for confirmation from the keynote speakers, then I'll merge

File: index.html [hunk 1]
```diff
@@ -267,64 +267,115 @@ <h1>Live video stream on the talks</h1>
...
+		  </div>
+		  <div class="speaker-social">
+		    <a href="https://r-forge.r-project.org/users/mmaechler/"><img src="images/social/rforge.png" style="height:20px;"></a>
+		    <a href="https://github.com/mmaechler"><img src="images/social/gh.png"></a>
+		    <a href="https://twitter.com/mmaechler"><img src="images/social/tw.png"></a>
+                    <a href="http://stat.ethz.ch/~maechler/"><img src="images/social/rss.png"></a>
+                    <a href="https://www.r-pkg.org/search.html?q=Martin+Maechler"><img src="images/social/R.png"></a>
+		  </div>
+		  <div class="speaker-bio">
+		    Martin is a Mathematician and Statistician, Professor of ETH Zurich, R Core member, Secretary General of the R Foundation, authored more than 20 R packages (such as Matrix, cluster, robustbase, cobs, VLMC, bitops or copula), Emacs ESS Core Developer since 1997 and Project Leader since 2004, author of several books and over 50 scientific journal articles.
```
Comment (mmaechler) [inline index.html]: Mathematician **(Ph.D. ETH Z)** and Statistician, Lecturer and Senior Scientist at Seminar für Statistik, ETH Zurich, ...

File: index.html [hunk 2]
```diff
@@ -267,64 +267,115 @@ <h1>Live video stream on the talks</h1>
...
+
+		<div class="span4 speaker">
+		  <div class="speaker-photo">
+		    <img src="images/speakers/martin_maechler.png" />
+		  </div>
+		  <div class="speaker-name">
+		    Martin Mächler
+		  </div>
+		  <div class="speaker-affiliation">
+		    Professor of Mathematics<br>at ETH Zurich (CH)
```
Comment (mmaechler) [inline index.html]: Senior Scientist  in Statistics (Mathematics)

**Comment by daroczig** [comment]
Thank you very much, @mmaechler, just updated the PR -- will merge after some finishing touches (hopefully) tomorrow.

**Pull request merged**